src/struct_info.json

```diff
@@ -1227,6 +1227,12 @@
                 "size",
                 "mappedAtCreation"
             ],
+            "WGPUBufferMapCallbackInfo": [
```

Comment:
I assume this should not be there because `MapAsyncF` is not as well?
Does this means all Future related stuff below should be removed also.

Reply:
I chose not to remove those things adding them back will increase the size of the next roll. So I just removed the minimum needed to make it build (I did this manually, not in dawn.json).